kythe/java/com/google/devtools/kythe/extractors/java/standalone/aspect.bzl

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//tools/build_rules/verifier_test:verifier_test.bzl", "extract")
```

Comment:
We should probably move extract out of verifier_test.bzl and to its own file/package if it will be used in these aspects but we can do that in a clean up.

Reply:
I'd really rather avoid scattering .bzl files throughout the code base; there are a few outside of `//tools` now, but I'd like to keep that to a minimum.

Reply:
Yeah, that makes sense. My thought was just that if extract is going to be used for more than just verification, it might help to put it in something like extraction.bzl instead of verifier_test.bzl